.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/telescope
+    docker:
+      - image: circleci/node:10
+      - image: circleci/redis:5 
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: npm ci
+      - run:
+          name: 
+          command: npm run build --if-present
```

Comment:
We currently do not have a build script, I don't think so we should add a step for it now.
I know it wont be executed, but it can be added later when we add a build script to our package.json.